src/Primitives/Container.purs

```diff
@@ -336,28 +352,22 @@ getContainerProps :: ∀ o item e.
      (H.IProp
         ( onMouseDown :: ET.MouseEvent
         , onMouseUp :: ET.MouseEvent
-        , onBlur :: ET.FocusEvent
```

Comment:
👍 Glad to see these events being removed. Less is better as far as listeners go.